src/Azure.Functions.Cli/Secrets/UserSecretsConfigurationBuilder.cs

```diff
@@ -10,8 +9,9 @@ namespace Azure.Functions.Cli.Diagnostics
     internal class UserSecretsConfigurationBuilder : IConfigureBuilder<IConfigurationBuilder>
     {
         private readonly string _scriptPath;
+        private readonly LoggingFilterHelper _loggingFilterHelper;
 
-        public UserSecretsConfigurationBuilder(string scriptPath)
+        public UserSecretsConfigurationBuilder(string scriptPath, LoggingFilterHelper loggingFilterHelper)
```

Comment:
@gzuber - can you please ACK these changes? Had to re-base and update this code so that all the logs go through default logging filters